format/index.md

```diff
@@ -873,15 +880,22 @@ for each relationship name can be specified:
 GET /articles/1?include=comments.author
 ```
 
-> Note: A request for `comments.author` should not automatically also
-include `comments` in the response. This can happen if the client already
-has the `comments` locally, and now wants to fetch the associated authors
-without fetching the comments again.
+> Note: Because compound documents require full linkage, intermediate
+resources in a multi-part path must be returned along with the leaf nodes.
+For example, a response to a request for `comments.author` should
+automatically include `comments` as well as the `author` of each of those
+`comments`.
+
+> Note: A server may choose to expose a deeply nested relationship such as
```

Comment:
Ok, +1. That text makes perfect sense now! (And it probably made sense before, I just misread it.)